cts/cli/regression.rules.exp

```diff
@@ -36,6 +36,8 @@ log_xmllib_err 	error: XML Error: ^
 log_xmllib_err 	error: XML Error: Entity: line 1: parser error : Start tag expected, '<' not found
 log_xmllib_err 	error: XML Error: invalidxml
 log_xmllib_err 	error: XML Error: ^
+pcmk__xml_parse_string 	error: Couldn't parse XML from string: Start tag expected, '<' not found
+ | (domain=1, level=3, code=4)
```

Comment:
Yeah I'm inclined to agree. IIRC these new errors are from calling an error logger consistently when we previously did it only for certain errors. (As I'm typing this, I'm not in a position to look over it)

Reply:
Dropping

Reply:
We could keep the "Couldn't parse XML from ..." part; might be helpful since `pcmk__log_xmllib_error()` doesn't tell us what we were trying to parse.

If so, there's the question of whether to log the unparsable input from stdin, since it's ephemeral.

Reply:
I don't think we need it

Reply:
I'll drop it then. I was thinking this left the user with absolutely no indication of what was invalid, but it's just "invalidxml" in this test, which is displayed.

Can't remember if anything will tell the user the name of the file that can't be parsed, if it's a filename instead of a string or stdin. I guess that's okay.